### PR TITLE
[stable/influxdb] Fix backups to GCS

### PR DESCRIPTION
**Pull request by mauermann**
#### What this PR does / why we need it:
Fix backups to GCS

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/influxdb/templates/backup-cronjob.yaml [hunk 1]
```diff
@@ -62,9 +62,9 @@ spec:
               - name: SRC_URL
                 value: /backups
               - name: DST_URL
-                value: {{ .Values.gcs.backup.destination}}
+                value: {{ .Values.backup.gcs.destination}}
               - name: KEY_FILE
-                value: /var/secrets/google/key.json
+                value: /var/secrets/google/{{ .Values.backup.gcs.serviceAccountSecretKey }}
```
Comment (Aisuko) [inline stable/influxdb/templates/backup-cronjob.yaml:67]: Could you please explain more details about this?

**Review by Aisuko — CHANGES_REQUESTED** [review]
Thanks for your PR

File: stable/influxdb/templates/backup-cronjob.yaml (hunk 1 above)
Comment (mauermann) [reply stable/influxdb/templates/backup-cronjob.yaml:67]: We already have some existing service account secrets with the key field "credentials.json". I'm not sure why we use this name, but IMHO this is/was the default file name some apps/libs expect.
We also use https://github.com/kiwigrid/gcp-serviceaccount-controller, this creates secret with credentials.json field name too.

**Review by monotek — APPROVED** [review]
/lgtm

**Comment by Aisuko** [comment]
Please resolve the conflict. Thanks.

File: stable/influxdb/templates/backup-cronjob.yaml (hunk 1 above)
Comment (Aisuko) [reply stable/influxdb/templates/backup-cronjob.yaml:67]: Thanks

**Review by Aisuko — APPROVED** [review]
/lgtm

**Pull request merged**